Sources/PointFree/Account/Account.swift

```diff
@@ -483,8 +483,8 @@ private let subscriptionTeammateOverview = View<AccountData> { data -> [Node] in
 
 private func planName(for subscription: Stripe.Subscription) -> String {
   return subscription.quantity > 1
-    ? subscription.plan.name + " (×" + String(subscription.quantity) + ")"
-    : subscription.plan.name
+    ? subscription.plan.nickname + " (×" + String(subscription.quantity) + ")"
```

Comment:
I've utilized the optional `nickname` field for display purposes and ensured that it's set for all of our plans.